tests/unit/k8s/K8sInterface_test.cpp

```diff
@@ -47,9 +45,7 @@ TEST_F(K8sInterfaceTest, TestGetServiceList) {
 }
 
 TEST_F(K8sInterfaceTest, TestCreateJasmineGraphWorkerDeployment) {
-    v1_node_list_t *nodeList = interface->getNodes();
-    auto nodeName = static_cast<v1_node_t *>(nodeList->items->firstEntry->data)->metadata->name;
-    v1_deployment_t *deployment = interface->createJasmineGraphWorkerDeployment(1, "10.43.0.1", nodeName);
+    v1_deployment_t *deployment = interface->createJasmineGraphWorkerDeployment(1, "10.43.0.2", "10.43.0.1");
```

Comment:
Why have we hard coded IP address there?

Reply:
They are dummy IP addresses (Yet in Kubernetes default IP address range) for test cases to check worker deployment creations.